CraftsmenLtd/BloodConnect
# I#261: Added Terms of Service and Privacy Policy Links functionality

Fixes #261


## What kind of change does this PR introduce (check at least one)


## How would you describe the changes in a changelog (1-2 lines)?

Added Terms of Service and Privacy Policy links functionality to the adding personal information page, enabling users to view legal documents directly from the app using device default browser.

## How to test


Test steps:
1. Navigate to the personal information page
2. Verify Terms of Service and Privacy Policy links are visible
3. Tap on each link and confirm they open the correct URLs
4. Ensure links are functional and redirect to the correct documents

## Checklist


## Additional Notes (optional)

The Terms of Service and Privacy Policy links are now pointing to static HTML pages from our BloodConnect project's documentaion hosted on the project's GitHub Pages.

## Deploy risk & follow up
